src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Blocks/customizations.html.twig

```diff
@@ -0,0 +1,7 @@
+{% form_theme productForm.customizations '@PrestaShop/Admin/Sell/Catalog/Product/Blocks/customizations_theme.html.twig' %}
+
+{% block form_product_customizations %}
+  <div id="productCustomizations">
```

Comment:
```suggestion
  <div id="product-customizations">
```

I feel like we don't use camelCase anywhere in classes and ids, maybe I'm wrong ?

Reply:
you are right 
https://github.com/PrestaShop/PrestaShop/issues/15977#issuecomment-707563757